src/v2/Components/UserSettings/UserEmailPreferences/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react"
+import { Banner, Box, Flex, SelectSmall, Serif } from "@artsy/palette"
+import { useSystemContext } from "v2/Artsy/SystemContext"
+import { QueryRenderer, graphql } from "react-relay"
```

Comment:
We should be using `SystemQueryRenderer` here, which guards against server-side fetches